pipeline/extract/gog/extract_gog.py

```diff
@@ -12,6 +12,8 @@
 
 
 PLATFORM_TITLE = 'window.productcardData.cardProductSystemRequirements'
+FILTER_1 = 'This Game may contain content not appropriate for all ages or may not be appropriate for viewing at work'
```

Comment:
Good use of global variables as your filters; makes it easy to change if GOG ever do so in the future